src/Header.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import './App.css';
+import logo from "./images/logo3.png";
+
+
+class Header extends Component {
+  render() {
+    return (
+      <div>
+      <div className="col-md-9 col-xs-9">
```

Comment:
It's been a long time since I used bootstrap, but is there a reason you need to specify col-md and col-xs? I thought col-xs values applied to the smallest screens and anything larger?